docker-compose.yaml

```diff
@@ -64,8 +67,15 @@ services:
       - BANK_URL=http://bank:8070
       - DB_FILE=/data/my.db
       - LOG_LEVEL=debug
-      - OTP_MODE=mock
-      # - MOBILE_CLAIM=mobile_verified
+      - GIN_MODE=debug # change to release to disable gin debug
```

Comment:
Can we leave it commented out?

Reply:
debug is there by default 

```
func init() {
	mode := os.Getenv(EnvGinMode)
	SetMode(mode)
}

// SetMode sets gin mode according to input string.
func SetMode(value string) {
	if value == "" {
		value = DebugMode
	}
```

I wanted it to be more explicit that's why I've placed that in `docker-compose`